docs/content/en/docs/yaml-crd-ref/taskdefinition.md

```diff
@@ -160,7 +238,7 @@ and the revised code is picked up without additional changes.
 
 ## Examples
```

Comment:
Does it make sense if we separate examples under each runtime? 
e.g.

- task definition generic explanation
- python runtime
  - info on how the task def looks like
  - examples
- js deno runtime
  - info on how the task def looks like
  - examples
- container runtime
   - info on how the task def looks like
  - examples

Reply:
Yes, this makes a lot of sense...

Reply:
This is done although I'm not sure I have the titles correct.